### PR TITLE
Failing test case for #103

### PR DESCRIPTION
**Pull request by rvignesh89**
For #103

### CODE REVIEW

File: dynamic_preferences/api/viewsets.py [hunk 1]
```diff
@@ -162,6 +162,11 @@ def get_manager(self):
             instance=self.get_related_instance()
         )
 
+    def get_queryset(self):
+        return super().get_queryset().filter(
```
Comment (EliotBerriot) [inline dynamic_preferences/api/viewsets.py]: I think you need to call `super(PerInstancePreferenceViewSet, self)` to fix those failing tests and ensure compatibility with Python 2 :)

**Comment by EliotBerriot** [comment]
Actually, I'm sorry about the failing tests, this is because the latest version of DRF is not compatible with django <1.10 anymore.

Once #107  is merged, you can rebase your branch on develop to ensure your test pass as expected.

**Comment by rvignesh89** [comment]
Oh I see. Alright then I'll wait for you to fix that.

**Comment by EliotBerriot** [comment]
it's okay you can now rebase :)

**Comment by rvignesh89** [comment]
Alright, looks good now!

**Pull request merged**